vnext/ReactCommon/ReactCommon.vcxproj

```diff
@@ -182,6 +182,7 @@
     <ClInclude Include="$(ReactNativeDir)\ReactCommon\react\renderer\runtimescheduler\RuntimeScheduler_Modern.h" />
     <ClInclude Include="$(ReactNativeDir)\ReactCommon\react\renderer\core\PropsParserContext.h" />
     <ClCompile Include="$(ReactNativeDir)\ReactCommon\react\bridging\LongLivedObject.cpp" />
+    <ClInclude Include="$(ReactNativeDir)\ReactCommon\react\bridging\EventEmitter.h" />
```

Comment:
Don't add this ClInclude in the middle of ClCompiles. Add EventEmitter.h under CallbackWrapper.h with the other ClIncludes.